src/render/tree/mod.rs

```diff
@@ -89,6 +89,9 @@ impl Tree {
                 let mut root_id = None;
 
                 while let Ok(TraversalState::Ongoing(node)) = rx.recv() {
+                    if ctx.dirs_only && !node.is_dir() {
```

Comment:
Hmm so doing it here would result in the loss of disk usage information which is something I would still want to compute and surface to the end-user unless they supply the `--supress-size` option. We'd probably want to filter for dirs on the display step which happens [somewhere here](https://github.com/solidiquis/erdtree/blob/3448b39cae96155d67454e9c12e8303fe4576a07/src/render/tree/mod.rs#L268).

Reply:
Sounds reasonable, I'll update the PR accordingly.

Reply:
Just saw that this should have printed `└─` in front of `utils`.

Reply:
There isn't currently a way to disable colors. Feel free to open an issue for that. I'd likely be the one to address that as unfortunately given how erdtree is built it would require a lot of tedious changes.

Reply:
I think we need to filter this on a node and not on a display level.  I'll notify you when I have a proper fix.

Reply:
@solidiquis It's not ideal, but pruning the file nodes after the tree has been assembled seems to be the easiest way to get a dirs-only view that also includes size information.  To get directory sizes we need to accumulate the size of all nodes below the directory.  Commit 3333586 contains the patch that prunes the file nodes if `dirs-only` is set.

Reply:
Also, the tree is now properly rendered:

<img width="516" alt="Screenshot 2023-03-23 at 21 19 17" src="https://user-images.githubusercontent.com/1538057/227345081-d2f68d74-2c89-42f1-a93e-737736d00e46.png">